### ISSUE TITLE
Indicate hoisted item with label or breadcrumb

### ISSUE THREAD

**Issue by jessegrosjean** [FoldingText/foldingtext-for-atom#10]
I do think there should be some option for this, but I'm not sure it should be part of the core outliner. I personally like NOT seeing the hoisted item. It's also not clear if there's a best place to put it:
1. Above outliner – Probably most standard place to put this. But I don't think can be implemented well as an atom package, and would be redundant if any of the following solutions are used.
2. In status bar – Keeps outline view clean, but maybe hard to see, and there might be other items in the status bar, so room could be an issue. Can be implemented as a separate package.
3. In side bar, show tree view of all items with children, and highlight the one that's currently hoisted (making FoldingText into a two pane outliner). – Clearest and makes hoisting easiest (just click item), bu takes up a lot of space. Can be implemented as a separate package.

**Comment by derekvan** [comment]
A few ideas:

Mainly what I'm trying to replicate is FoldingText's current (not the Atom version) capability of "focusing" on a heading, which leaves the focused heading and contents and folds everything else away. E.g.,

![screen shot 2015-05-18 at 10 48 55 am](https://cloud.githubusercontent.com/assets/890341/7683222/8b71cb62-fd4b-11e4-9be7-34b57a6e696c.png)

So, could this be done via CSS/LESS? Show parent header if hoisted?

I'm not as taken by item 2 above because it wouldn't be very noticeable, but items 1 or 3 seem good. I wouldn't mind the space required by item 3, and in fact I might appreciate the tree view being present throughout just to keep tabs on the outline. However, this might also conflict with my tendency to use the "tree view" package to view file folder structure in a left pane.

**Comment by jessegrosjean** [comment]
Right… I like this feature better if it (as you suggest) just means "show current hoisted heading" instead of show entire breadcrumb path to root. In that case I could also replace that top level items triangle handle with a back arrow. Click on it to un-hoist.

For the next release my focus is on import/export of various file formats. But these are good notes for when I get to this feature. Thanks.
